Don't use tagfiles, this makes doxygen segfault every other run

--- cmake/doxygen/Doxyfile.in~	2018-05-13 15:39:08.000000000 +0200
+++ cmake/doxygen/Doxyfile.in	2018-07-19 15:04:46.420374326 +0200
@@ -1318,7 +1318,7 @@
 # If a tag file is not located in the directory in which doxygen
 # is run, you must also specify the path to the tagfile here.
 
-TAGFILES               = @DOXYTAG_ENTRIES@
+TAGFILES               = 
 
 # When a file name is specified after GENERATE_TAGFILE, doxygen will create
 # a tag file that is based on the input files it reads.
